src/python/gudhi/representations/metrics.py

```diff
@@ -50,9 +50,9 @@ def _compute_persistence_diagram_projections(X, num_directions):
         num_directions (int): number of lines evenly sampled from [-pi/2,pi/2] in order to approximate and speed up the distance computation.
 
     Returns: 
-        list of n numpy arrays of shape (2*numx2): list of projected persistence diagrams.
+        list of n numpy arrays of shape (2*numxnum_directions): list of projected persistence diagrams.
     """
-    thetas = np.linspace(-np.pi/2, np.pi/2, num=num_directions+1)[np.newaxis,:-1]
+    thetas = np.linspace(-np.pi/2, np.pi/2, num=num_directions)[np.newaxis,:-1]
```

Comment:
As is, I think thetas contains both -pi/2 and +pi/2. If that's redundant, we could pass `endpoint=False`.